packages/devtools_app/lib/src/debugger/evaluate.dart

```diff
@@ -296,7 +296,8 @@ Future<List<String>> autoCompleteResultsFor(
         );
         // TODO(grouma) - This shouldn't be necessary but package:dwds does
         // not properly provide superclass information.
-        result.addAll(instance.fields.map((field) => field.decl.name));
+        result.addAll(instance.fields.map((field) => field.decl.name).where(
+            (member) => !_isAccessible(member, instance.classRef, controller)));
```

Comment:
Should this be negated? Wouldnt we want to add results that are accessible?

Reply:
Yup! A test caught it thankfully. I also expanded this with another test. Also there was surprisingly a runtime type error.